### PR TITLE
Support for SDK 2.2

### PR DESCRIPTION
**Pull request by eskaur**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,12 +146,12 @@ Please visit [Zivid Knowledge Base](http://help.zivid.com) for general informati
 
 | Operating System | Python version     | Zivid SDK version |
 | :--------------- | :------------------| :---------------- |
-| Ubuntu 20.04     | 3.8                | 2.1.0             |
-| Ubuntu 18.04     | 3.6                | 2.1.0             |
-| Ubuntu 16.04     | 3.5                | 2.1.0             |
-| Fedora 30        | 3.7                | 2.1.0             |
-| Windows 10       | 3.6, 3.7, 3.8, 3.9 | 2.1.0             |
-| Arch Linux*      | latest             | 2.1.0             |
+| Ubuntu 20.04     | 3.8                | 2.2.0             |
```
Comment (nedrebo) [inline README.md:149]: Unrelated: We should include a 3.9 distro, maybe Fedora 33 is an easy win if we just base it on Fedora 30 work. Nice to add more non-Ubuntus to increase the test surface of Zivid SDK.

**Review by nedrebo — COMMENTED** [review]
Random comment, not full review:

File: README.md (hunk 1 above)
Comment (eskaur) [reply README.md:149]: Noted. Won't do it in this PR though.

**Review by trym-b — APPROVED** [review]
Excellent PR.

**Pull request merged**